package.json

```diff
@@ -47,6 +47,7 @@
     "html-loader": "0.4.3",
     "html-webpack-plugin": "2.22.0",
     "http-proxy-middleware": "0.17.1",
+    "intl-messageformat-parser": "1.4.0",
```

Comment:
ok, so we need both a string fix and a nodejs-modules build before merging :)

Reply:
Not for this one.  But I did upgrade the jest version subsequentially, so that may require a nodejs-modules dance.